assets/app/views/create.html

```diff
@@ -10,156 +10,199 @@
-
-                <div ng-if="emptyCatalog && loaded && !nonBuilderImages.length" class="empty-state-message empty-state-full-page">
-                  <h2 class="text-center">No images or templates.</h2>
-
-                  <p class="gutter-top">
-                    No images or templates are loaded for this project or the shared
-                    <code>openshift</code> namespace. An image or template is
-                    required to add content.
-                  </p>
-
-                  <p>
-                    To load an image stream or template from a file, run the command
-                    <div><code>oc create -f &lt;filename&gt; -n {{projectName}}</code></div>
-                  </p>
-
-                  <p><a href="{{projectName | projectOverviewURL}}">Back to overview</a></p>
-                </div>
-
-                <!-- Show an abbreviated message if only non-builders exist in the project.
-                     We show a link to view the non-builder images below. -->
-                <p ng-if="emptyCatalog && loaded && nonBuilderImages.length">No builder images or templates.</p>
-
-                <div ng-show="!emptyCatalog">
-                  <h1>Select Image or Template</h1>
-
-                  <div class="row">
-                    <!-- Right padding to line up cleanly with the left column tiles. -->
-                    <div class="col-sm-6 catalog-header-left">
-                      Choose from web frameworks, databases, and other components to
-                      add content to your project.
-
-                      <div class="filter-group">
-                        <!-- Add a hidden label for screen readers. -->
-                        <label for="search" class="sr-only">Filter by keyword</label>
-                        <!-- Pull the filter right so the input and dropdown button
-                             can use 100% width on the same line. -->
-                        <div uib-dropdown uib-keyboard-nav class="btn-group pull-right">
-                          <button class="btn btn-default dropdown-toggle" data-toggle="dropdown" role="menu">
-                            Browse
-                            <span class="caret" aria-hidden="true"></span>
-                          </button>
-                          <ul class="uib-dropdown-menu">
-                            <li ng-repeat="tag in browseGeneral" role="menuitem">
-                              <a href="" ng-click="filter.tag = tag">{{tag}}</a>
-                            </li>
-                            <li class="divider"></li>
-                            <li ng-repeat="tag in browseTechnologies" ng-if="tag" role="menuitem">
-                              <a href="" ng-click="filter.tag = tag">{{tag}}</a>
-                            </li>
-                          </ul>
+                <h1>Add Components to your Project</h1>
+
+                <uib-tabset justified="true">
```

Comment:
Well there was a discussion and I know that @rhamilto suggested to not have it justified. But then @ajacobs21e supplied the wireframes for the new create page and there it was justified, so I've figured out I will leave it justified and lets discuss it on the PR, here :)